amalisov.cuibono/webapp/controller/EditBonusTranche.controller.ts

```diff
@@ -76,12 +78,13 @@ export default class EditBonusTranche extends BaseController {
 						location: oData.location,
 						startDate: !isDuplicate ? oData.startDate : "",
 						endDate: !isDuplicate ? oData.endDate : "",
-						originDate: !isDuplicate ? oData.originDate : "",
+						orignDate: !isDuplicate ? oData.orignDate : "",
```

Comment:
the backend is the one with the typo so we told them to change it, but they replied it could require much process with the deployment included so we adjusted as well.